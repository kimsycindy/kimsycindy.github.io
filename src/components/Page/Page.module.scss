@import '../../styles/variables.scss';

.Page {
  display: block;
  flex: 1;
  position: relative;
  padding: $containerPadding;
}
