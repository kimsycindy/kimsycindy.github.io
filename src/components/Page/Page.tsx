import React from 'react'
import cx from 'classnames'

import styles from './Page.module.scss'

interface Props {
  className?: string
}

const Page: React.FC<Props> = ({ children, className }) => (
  <div className={cx(styles.Page, className)}>{children}</div>
)

export default Page
