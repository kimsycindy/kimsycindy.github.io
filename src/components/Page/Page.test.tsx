import React from 'react'
import renderer from 'react-test-renderer'

import Page from './Page'

describe('Page', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Page className="page-mock">
          <div>Page</div>
        </Page>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
