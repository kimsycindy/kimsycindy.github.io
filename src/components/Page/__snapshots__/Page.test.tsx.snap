// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders correctly 1`] = `
<div
  className="Page page-mock"
>
  <div
    className="Container undefined"
  >
    <div>
      Page
    </div>
  </div>
</div>
`;
